docs/tutorials/add-a-user_dmanis.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+---
+
+# Tutorial: Add a user
+
+This tutorial will teach you how to add a new user to the To-Do system.
+
+Allow 30 minutes to complete this lesson.
+
+## Prerequisites
+
+* The To-Do service is up and running. Follow [the setup procedure](../before-you-start-a-tutorial.md) to install and run the To-Do service on your local system.
+* You are comfortable working with [REST interfaces](https://restfulapi.net).
+* A REST client such as [cURL](https://curl.se), [Postman](https://www.postman.com), or [Insomnia](https://insomnia.rest). This tutorial assumes you are using cURL but use whatever REST client you like.
+
+## Steps
```

Comment:
This section would be a good place to describe the procedures that follow and how they contribute to the overall goal.

Then make each procedure an H3 (`###`) with a title that describes the procedure goal (as the following sections do).